### PR TITLE
standalone sparqlwrapper

### PR DESCRIPTION
**Pull request by satra**
replacement sparqlwrapper

### CODE REVIEW

**Comment by satra** [comment]
@synapticarbors - any ideas here?

**Comment by satra** [comment]
@synapticarbors - could this be merged?

**Comment by synapticarbors** [comment]
@satra Sorry, I was out-of-town and just got back. I wonder if the conda-forge linter is failing since your recipe is named `meta.yml` rather than `meta.yaml`. Could you rename the file and see if that fixes the issue?

**Comment by satra** [comment]
@synapticarbors - i can try to but i suspect it comes from that previous merge for keepalive that had this recipe in it at one point.

**Comment by satra** [comment]
@synapticarbors - done. hopefully the other checks pass soon.

File: recipes/sparqlwrapper/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - keepalive
```
Comment (synapticarbors) [inline recipes/sparqlwrapper/meta.yaml]: Is `keepalive` a `build` req?

**Comment by satra** [comment]
@synapticarbors - it seems that issues are fixed now.

**Comment by satra** [comment]
done here: https://github.com/RDFLib/sparqlwrapper/pull/76

File: recipes/sparqlwrapper/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+test:
+  imports:
+    - SPARQLWrapper
+
+about:
+  home: https://github.com/RDFLib/sparqlwrapper
+  license: W3C License
+  # 'LICENSE file is missing in source due to the project missing a MANIFEST.in to copy into the release.'
```
Comment (satra) [inline recipes/sparqlwrapper/meta.yaml]: https://github.com/RDFLib/sparqlwrapper/pull/76

**Comment by jakirkham** [comment]
Closing/reopening to get the linter.

**Comment by satra** [comment]
@jakirkham - can this be merged?

**Comment by satra** [comment]
@jakirkham - any idea why this feedstock is empty:

https://github.com/conda-forge/sparqlwrapper-feedstock

**Pull request merged**